src/db/spanner/batch.rs

```diff
@@ -472,6 +491,11 @@ pub async fn do_append_async(
             .param_types(param_types.clone())
             .execute_dml_async(&db.conn)
             .await?;
+            db.metrics.count_with_tags(
```

Comment:
this emits within the update loop (one per update) and always emits the same count_update (the number of total updates). and it's the same count emitted above (existing.len() as `storage.spanner.batch.pre-existing`). so I think this should just be killed?

Reply:
I should move it out of the update loop ('doh) but I was thinking more that `exists` is whatever was already in the table and this was the number of elements that resulted in updates, because a batch could self update as  well  (rare, but potential).

Reply:
If anything I was debating killing the `metrics.incr` above, since it's fairly low information